package.json

```diff
@@ -42,6 +42,7 @@
     "js-file-download": "^0.4.12",
     "marked": "^0.7.0",
     "nuxt": "^2.12.1",
+    "nvm": "0.0.4",
```

Comment:
do we actually need this dependecy?

Reply:
I needed to use nvm to switch to an earlier version (12.13.1), but I dont imagine that it's necessary in the general case